lib/scottkit/game.rb

```diff
@@ -104,6 +110,15 @@ def initialize(options)
       @options = options
       @rooms, @items, @actions, @nouns, @verbs, @messages =
         [], [], [], [], [], []
+      @output = options[:output] || $stdout
+    end
+
+    def puts *args
+      output.puts(*args)
```

Comment:
I'm not sure about overriding `print` in `game.rb`. Isn't the more honest approach you used in the previous commit also suitable here?

Reply:
Oh ... but you're not _overriding_ `print`, are you? You're just defining a method with the same name as the underlying primitive that it wraps -- so the invocation changes from `print(x)` to `@game.print(x)`. OK, that makes perfect sense. Carry on  :-)

Reply:
Yeah kind of funky to have to do this but it's either this or passing the output object down to the instructions. This seemed cleaner and having it centralized would give you one place to hook in a driver.

Reply:
Yep, this is good.